addons/a11y/src/components/ColorBlindness.tsx

```diff
@@ -1,5 +1,5 @@
 import { document } from 'global';
-import React, { ReactNode, useState } from 'react';
+import React, { FunctionComponent, ReactNode, useState } from 'react';
```

Comment:
With `import React, { FunctionComponent, ReactNode, useState } from 'react';` we are still importing React using default syntax I think you should use `import * as React from 'react';` and `React.FunctionComponent` etc everytime `import React from 'react';` is still needed.

Reply:
hmmm, in fact, is importing React still needed?

Reply:
It's indeed needed for the TSX files.